operators/pkg/instance-controller/common_logic.go

```diff
@@ -13,19 +13,28 @@ import (
 )
 
 // CreateInstanceExpositionEnvironment creates the components necessary to access the environment (service, ingress and oauth2-proxy related resources).
+// Additionally, it makes the service expose another port and creates an ingress for file manager sidecar container (only for container environments).
 func (r *InstanceReconciler) CreateInstanceExpositionEnvironment(
 	ctx context.Context,
 	instance *crownlabsv1alpha2.Instance,
-	name string,
-) (v1.Service, networkingv1.Ingress, error) {
+	name string, hasFileManager bool,
```

Comment:
```suggestion
	name string, hasFileBrowser bool,
```

I would name it *file browser* instead of *file manager*, for the sake of clarity. The same for all the other references.